docs/application/dotnet/guides/account/data-sync.md

```diff
@@ -86,32 +86,32 @@ To set callbacks in your Sync Adapter service application that your UI applicati
 
     When a specific event is detected or a sync job is requested, the `StartSyncCallback()` or `CancelSyncCallback()` callbacks are invoked.
 
-    ```
+    ```csharp
     SyncAdapter obj = new SyncAdapter();
     obj.SetSyncEventCallbacks(StartSyncCallback, CancelSyncCallback);
     ```
 
 3.  When the sync operations are no longer needed, unset the callbacks to free the `Tizen.Account.SyncManager.SyncAdapter` instance:
 
-    ```
+    ```csharp
     obj.UnsetSyncEventCallbacks();
     ```
 
 <a name="set_parameters"></a>
 ## Defining a Sync Job
```

Comment:
```suggestion 
## Define sync job

Reply:
Done. 'Define Sync Job' was applied. Without applying 'Sentence case' - according to our approach https://github.com/Samsung/tizen-docs/pull/1387#discussion_r575045312